e2e/glustershd_test.go

```diff
@@ -47,12 +54,55 @@ func testSelfHeal(t *testing.T, tc *testCluster) {
 
 	r.Nil(client.VolumeStart(vol1.Name, false), "volume start failed")
 
+	checkFuseAvailable(t)
+
+	mntPath := testTempDir(t, "mnt")
+	defer os.RemoveAll(mntPath)
+
+	host, _, _ := net.SplitHostPort(tc.gds[0].ClientAddress)
+	err = mountVolume(host, volname, mntPath)
+	r.Nil(err, fmt.Sprintf("mount failed: %s", err))
+
+	defer syscall.Unmount(mntPath, syscall.MNT_FORCE)
+
+	f, err := os.Create(mntPath + "/file1.txt")
+	r.Nil(err, fmt.Sprintf("file creation failed: %s", err))
+	f.Close()
+
+	getBricksStatus, err := client.BricksStatus(volname)
+	r.Nil(err, fmt.Sprintf("brick status operation failed: %s", err))
+	for brick := range getBricksStatus {
+		if getBricksStatus[brick].Info.PeerID.String() == tc.gds[0].PeerID() {
+			process, err := os.FindProcess(getBricksStatus[brick].Pid)
+			r.Nil(err, fmt.Sprintf("failed to find bricks pid: %s", err))
```

Comment:
This test will be silently ignored if condition in line 75 is not satisfied. Add a test before for loop to check required number of BricksStatus with gds[0].PeerID